### PR TITLE
Update model possibilities

### PR DESCRIPTION
**Pull request by ellennickles**
Starting to refine and fill in the gaps in the “Create with RunwayML: Model Possibilities” section to prepare for upcoming classes that will use the app. Like last time, no links were modified with changes in this PR. 

My proposal for organizing models is not perfect. Some models can live in multiple categories, and I’m sure we can label sub-categories more accurately, but this is start. Now that all the models all here, we can easily move them around.

The table format is helpful for us and contributors to see at a glance where we can fill in with extra resources (demos, tutorials, lessons, etc.)

### CODE REVIEW

File: docs/_sidebar.md [hunk 1]
```diff
@@ -16,32 +16,15 @@
...
-            * [RunwayML Library](https://github.com/runwayml/processing-library)
-        * p5.js
-            * [Contour Drawings I](tutorials/tutorial_photosketch.md)
-            * [Contour Drawings II](tutorials/tutorial_p5_photosketch.md)
-            * [Images from Text](tutorials/tutorial_p5_attngan.md)
-            * [Image Translation](tutorials/tutorial_p5_cyclegan.md)
-        * More soon!
-    * 🔌**Plugins**
+        * [Integrations](networking/integrations.md)
+    * 🔌 **Plugins**
```
Comment (cvalenzuela) [inline docs/_sidebar.md]: I think the Processing library is in a good position to be listed here.

File: docs/create-with-runwayml/detect.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Detect
+
+ The pre-trained models in this category identify and label content. You'll find examples here for detecting objects, places, and people in real-time.
+ 
+ Note: models are added to the RunwayML app often, and this page does not necessarily respresent the full list. Contributions, demos, and tutorials are welcome here!
```
Comment (cvalenzuela) [inline docs/create-with-runwayml/detect.md]:
```suggestion
> **Note**: models are added to the RunwayML app often, and this page does not necessarily represent the full list. Contributions, demos, and tutorials are welcome here!
```

Comment (cvalenzuela) [reply docs/create-with-runwayml/detect.md]: This is a great point to add!

File: docs/create-with-runwayml/detect.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Classify
+📽 [How to Run Your First Image Classifier](https://www.youtube.com/watch?v=yoJWVSL1ST4)<br>
+📽 [How Neural Networks See the World](https://www.youtube.com/watch?v=MElaCVugavw)
+
+| Model | Description |  |
```
Comment (cvalenzuela) [inline docs/create-with-runwayml/detect.md]:
```suggestion
| Model | Description | More |
```

File: docs/create-with-runwayml/detect.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+| [MobileNet](https://open-app.runwayml.com/?model=runway/mobilenet) | Classify images in real-time | [Tutorial](tutorials/tutorial_mobilenet.md) | 
+| [Places365](https://open-app.runwayml.com/?model=matthewbay/places365) | Identify places (bedroom, dining room, mall, etc.)| |
+| [YOLACT](https://open-app.runwayml.com/?model=zaid/yolact) | Identify and segment objects in real-time | |
+| [Visual-Importance](https://open-app.runwayml.com/?model=runway/visual-importance) | Detect important areas in an image | |
+
+
+## Face/Body Tracking
+📽 [How to Use Motion Capture Models](https://www.youtube.com/watch?v=3q-HUG3C6IE)
+
+| Model | Description |  |
```
Comment (cvalenzuela) [inline docs/create-with-runwayml/detect.md]:
```suggestion
| Model | Description | More |
```

File: docs/create-with-runwayml/detect.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+| [3D Dense Face Alignment](https://open-app.runwayml.com/?model=matthewbay/3ddfa) | 3D face alignment | |
+| [Face-Landmarks](https://open-app.runwayml.com/?model=runway/face-landmarks) | Detect features on images of faces |  |
+| [OpenPifPaf](https://open-app.runwayml.com/?model=alexandre01/openpifpaf-pose) | Perform human pose estimation in crowded images |  |
+| [PoseNet](https://open-app.runwayml.com/?model=runway/posenet) | Perform real-time skeletal tracking of one or more people |  |
+
+
+## OCR
+Optical Character Recognition
+
+| Model | Description |  |
```
Comment (cvalenzuela) [inline docs/create-with-runwayml/detect.md]:
```suggestion
| Model | Description | More |
```

File: docs/create-with-runwayml/generate.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
+# Generate
+
+The pre-trained models in this category generate synthetic output based the training data, such as photorealistic images and coherent text.
+
+Note: models are added to the RunwayML app often, and this page does not necessarily respresent the full list. Contributions, demos, and tutorials are welcome here!
```
Comment (cvalenzuela) [inline docs/create-with-runwayml/generate.md]:
```suggestion
> **Note**: models are added to the RunwayML app often, and this page does not necessarily represent the full list. Contributions, demos, and tutorials are welcome here!
```

File: docs/create-with-runwayml/generate.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+## Generate Images from Text
+| Model | Description |  |
```
Comment (cvalenzuela) [inline docs/create-with-runwayml/generate.md]:
```suggestion
| Model | Description | More |
```

File: docs/create-with-runwayml/generate.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+| :--- | :---| :--- |
+| [AttnGAN](https://open-app.runwayml.com/?model=runway/AttnGAN) | Turn text descriptions of scenes into synthesized images | [Tutorial](tutorials/tutorial_t2i.md) |
+
+
+## Generate Text from Images
+| Model | Description |  |
```
Comment (cvalenzuela) [inline docs/create-with-runwayml/generate.md]:
```suggestion
| Model | Description | More |
```

File: docs/create-with-runwayml/generate.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+| :--- | :---| :--- |
+| [im2txt](https://open-app.runwayml.com/?model=runway/im2txt) | Generate sentence descriptions of images | [Tutorial](tutorials/tutorial_im2txt.md) |
+
+
+## Generate Text
+| Model | Description |  |
```
Comment (cvalenzuela) [inline docs/create-with-runwayml/generate.md]:
```suggestion
| Model | Description | More |
```

File: docs/create-with-runwayml/generate.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+| :--- | :---| :--- |
+| [GPT-2](https://open-app.runwayml.com/?model=runway/GPT-2) | Generate paragraphs of text from a prompt | |
+| [OpenGPT-2](https://open-app.runwayml.com/?model=runway/OpenGPT-2) | Replication effort of GPT-2's largest model (1.5B parameters) | |
+| [universal-sentence-encoder-xling-runway](https://open-app.runwayml.com/?model=aparrish/universal-sentence-encoder-xling-runway) | Encodes sentences to fixed-length vectors | |
+| [vae-lagging-encoder-poetry](https://open-app.runwayml.com/?model=aparrish/vae-lagging-encoder-poetry) | Gutenberg Poetry Variational Autoencoder | |
+
+
+## Synthesize Images
+| Model | Description |  |
```
Comment (cvalenzuela) [inline docs/create-with-runwayml/generate.md]:
```suggestion
| Model | Description | More |
```

File: docs/create-with-runwayml/transform.md [hunk 11]
```diff
@@ -0,0 +1,87 @@
+# Transform
+
+The pre-trained models in this category provide options to transform your original content in various ways. Though some models below are described with "generative" qualities, all models here modify input to produce a transformed output.
+
+Note: models are added to the RunwayML app often, and this page does not necessarily respresent the full list. Contributions, demos, and tutorials are welcome here!
```
Comment (cvalenzuela) [inline docs/create-with-runwayml/transform.md]:
```suggestion
> **Note**: models are added to the RunwayML app often, and this page does not necessarily represent the full list. Contributions, demos, and tutorials are welcome here!
```

File: docs/create-with-runwayml/transform.md [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+
+## Style Transfer
+| Model | Description |  |
```
Comment (cvalenzuela) [inline docs/create-with-runwayml/transform.md]:
```suggestion
| Model | Description | More |
```

File: docs/create-with-runwayml/transform.md [hunk 13]
```diff
@@ -0,0 +1,87 @@
...
+| [Adaptive Style Transfer](https://open-app.runwayml.com/?model=runway/Adaptive-Style-Transfer)| Repaint images in the styles of famous painters | [Tutorial](tutorials/tutorial_style_transfer.md) |
+| [AdaIN-Style-Transfer](https://open-app.runwayml.com/?model=reiinakano/AdaIN-Style-Transfer) | Real-time style transfer | | 
+| [Arbitrary-Style-Transfer](https://open-app.runwayml.com/?model=runway/Arbitrary-Image-Stylization) | Real-time style transfer | |
+| [Fast Photo Style](https://open-app.runwayml.com/?model=reiinakano/FastPhotoStyle) | A Closed-form Solution to Photorealistic Image Stylization | |
+| [Fast-Style-Transfer](https://open-app.runwayml.com/?model=genekogan/Fast-Style-Transfer) | Stylize images in the styles of famous painters | |
+| [Style2paints](https://open-app.runwayml.com/?model=zaid/style2paints) | Automatic Sketch Colorization | |
+
+
+## Image Translation
+| Model | Description |  |
```
Comment (cvalenzuela) [inline docs/create-with-runwayml/transform.md]:
```suggestion
| Model | Description | More |
```

File: docs/create-with-runwayml/transform.md [hunk 14]
```diff
@@ -0,0 +1,87 @@
...
+| [CycleGAN](https://open-app.runwayml.com/?model=reiinakano/CycleGAN) | Image-to-image translation without labeled data | |
+| [Pix2Pix](https://open-app.runwayml.com/?model=reiinakano/Pix2Pix) | Image-to-image translation with labeled data | |
+| [Pix2PixHD-Facemarks2Portrait](https://open-app.runwayml.com/?model=yining/pix2pixHD-Facemarks2Portrait) | Generate a portrait image from a face landmarks image | |
+| [UGATIT](https://open-app.runwayml.com/?model=runway/UGATIT) | Official Tensorflow implementation of U-GAT-IT: Unsupervised Generative Attentional Networks with Adaptive Layer-Instance Normalization for Image-to-Image Translation | |
+
+
+## Images → Semantic Image Maps
+Semantic Image Segmentation
+
+| Model | Description |  |
```
Comment (cvalenzuela) [inline docs/create-with-runwayml/transform.md]:
```suggestion
| Model | Description | More |
```

File: docs/create-with-runwayml/transform.md [hunk 15]
```diff
@@ -0,0 +1,87 @@
...
+| :--- | :---| :--- |
+| [DeepLab](https://open-app.runwayml.com/?model=genekogan/deeplab) | Extract semantic maps from objects in images | |
+| [DeepLabV3](https://open-app.runwayml.com/?model=runway/DeepLabV3) | Deep labeling for semantic image segmentation | |
+| [Face-Parser](https://open-app.runwayml.com/?model=anastasis/Face-Parser) | A pretrained model for parsing facial components from an image | |
+
+
+## Semantic Image Maps → Images
+Semantic Image Synthesis
+
+| Model | Description |  |
```
Comment (cvalenzuela) [inline docs/create-with-runwayml/transform.md]:
```suggestion
| Model | Description | More |
```

File: docs/create-with-runwayml/transform.md [hunk 16]
```diff
@@ -0,0 +1,87 @@
...
+| :--- | :---| :--- |
+| [SPADE-COCO](https://open-app.runwayml.com/?model=runway/spade-coco) | Transform semantic image maps into images | |
+| [SPADE-FACE](https://open-app.runwayml.com/?model=sree_harsha/spade-face) | Transform semantic image maps into images of faces| |
+| [SPADE-Landmarks](https://open-app.runwayml.com/?model=genekogan/SPADE-Landscapes) | Transform semantic image maps into landscape images | |
+
+
+## Image Editing
+| Model | Description |  |
```
Comment (cvalenzuela) [inline docs/create-with-runwayml/transform.md]:
```suggestion
| Model | Description | More |
```

File: docs/create-with-runwayml/transform.md [hunk 17]
```diff
@@ -0,0 +1,87 @@
...
+| [DeepFill](https://open-app.runwayml.com/?model=runway/DeepFill) | Remove objects from photos | |
+| [DeRaindrop](https://open-app.runwayml.com/?model=zaid/DeRaindrop) | Remove raindrops from photos| |
+| [Deep-Portrait-Image-Relighting](https://open-app.runwayml.com/?model=sree_harsha/Deep-Portrait-Image-Relighting) | Generate relighted image by using a single portrait image and a target lighting | |
+| [Edge-connect](https://open-app.runwayml.com/?model=zaid/edge-connect) | Generative Image Inpainting with Adversarial Edge Learning | |
+| [Image-Inpainting-GMCMM](https://open-app.runwayml.com/?model=anastasis/Image-Inpainting-GMCNN) | Image Inpainting via Generative Multi-column Convolutional Neural Networks | |
+| [PhotoSketch](https://open-app.runwayml.com/?model=runway/PhotoSketch]) | Turn images into contour drawings |  |
+
+
+## Colorize B/W Media
+| Model | Description |  |
```
Comment (cvalenzuela) [inline docs/create-with-runwayml/transform.md]:
```suggestion
| Model | Description | More |
```

File: docs/create-with-runwayml/transform.md [hunk 18]
```diff
@@ -0,0 +1,87 @@
...
+| :--- | :---| :--- |
+| [Automatic-Colorization](https://open-app.runwayml.com/?model=runway/Automatic-Colorization)| Colorize black and white images and videos | [Tutorial](tutorials/tutorial_colorizing_video.md)|
+| [DeOldify](https://open-app.runwayml.com/?model=reiinakano/DeOldify) | Colorize and restore old images | |
+
+
+## Upscale Images
+| Model | Description |  |
```
Comment (cvalenzuela) [inline docs/create-with-runwayml/transform.md]:
```suggestion
| Model | Description | More |
```

File: docs/create-with-runwayml/transform.md [hunk 19]
```diff
@@ -0,0 +1,87 @@
...
+| :--- | :---| :--- |
+| [ESRGAN](https://open-app.runwayml.com/?model=runway/ESRGAN) | 4x | [Tutorial](tutorials/tutorial_esrgan.md) |
+| [Image-Super-Resolution](https://open-app.runwayml.com/?model=runway/Image-Super-Resolution) | 2x | |
+
+## 2D Masking
+| Model | Description |  |
```
Comment (cvalenzuela) [inline docs/create-with-runwayml/transform.md]:
```suggestion
| Model | Description | More |
```

File: docs/create-with-runwayml/transform.md [hunk 20]
```diff
@@ -0,0 +1,87 @@
...
+| :--- | :---| :--- |
+| [MaskRCNN](https://open-app.runwayml.com/?model=runway/MaskRCNN) | Cut out objects from images | |
+| [Human-Segmentation](https://open-app.runwayml.com/?model=runway/Human-Segmentation) | Segment humans from images and videos | |
+
+
+## 3D Modeling
+| Model | Description |  |
```
Comment (cvalenzuela) [inline docs/create-with-runwayml/transform.md]:
```suggestion
| Model | Description | More |
```

File: docs/create-with-runwayml/generate.md [hunk 21]
```diff
@@ -0,0 +1,37 @@
+# Generate
+
+The pre-trained models in this category generate synthetic output based the training data, such as photorealistic images and coherent text.
```
Comment (cvalenzuela) [inline docs/create-with-runwayml/generate.md]:
```suggestion
The pre-trained models in this category can generate synthetic outputs based on specific training data, such as photorealistic images and coherent text.
```

File: docs/_sidebar.md (hunk 1 above)
Comment (ellennickles) [reply docs/_sidebar.md]: Yes! But I want to showcase that the other the networking-related examples (using Processing, p5.js, etc) in the networking section. Before "Integrations", I was using "Applications". Is that better? Or, maybe "Examples"?

Comment (cvalenzuela) [reply docs/_sidebar.md]: Yep, that sounds good. I think "Examples" works better

Comment (ellennickles) [reply docs/_sidebar.md]: great! all set!

**Pull request merged**